packages/destination-actions/src/destinations/moengage/trackEvent/index.ts

```diff
@@ -82,7 +82,14 @@ const action: ActionDefinition<Settings, Payload> = {
       default: {
         '@path': '$.properties'
       }
-    }
+    },    
+    update_existing_only: {
+      label: 'Update users from events',
+      type: 'boolean',
+      description: 'Settings to update the existing users through event sync',
+      required: false,
+      default: false
+    },
```

Comment:
```suggestion
    update_existing_only: {
      label: 'Update Existing Users Only',
      type: 'boolean',
      description: 'If set to true, events from the segment will only trigger updates for users who already exist in Moengage.',
      required: false,
      default: false
    },
```

@mishranik could you update the label and description to the suggestion above, in order that the behavior is more clear?